control/settings/controllers/settings.home.controller.js

```diff
@@ -57,16 +63,26 @@
           if (newObj.data && newObj.data.settings && newObj.data.settings.mapView == 'hide' && newObj.data.settings.defaultView == 'map') {
             SettingsHome.couponInfo.data.settings.defaultView = 'list';
             SettingsHome.showMessage = true;
-            setTimeout(function(){
+            setTimeout(function () {
               SettingsHome.showMessage = false;
               $scope.$digest();
-            },2000);
+            }, 2000);
           }
-          if(newObj.data) {
+          if (newObj.data && newObj.data.settings && newObj.data.settings.toggleEmployeeCode == 'on') {
```

Comment:
Why are we using string values ('on', 'off')? Instead of booleans (true, false)?

Reply:
Code formatter is disabled

Reply:
@gobelius